rust/main/hyperlane-core/src/types/message.rs

```diff
@@ -58,18 +57,7 @@ impl Default for HyperlaneMessage {
 
 impl Debug for HyperlaneMessage {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        write!(
-            f,
-            "HyperlaneMessage {{ id: {:?}, version: {}, nonce: {}, origin: {}, sender: {}, destination: {}, recipient: {}, body: 0x{} }}",
-            self.id(),
-            self.version,
-            self.nonce,
-            fmt_domain(self.origin),
-            fmt_address_for_domain(self.origin, self.sender),
-            fmt_domain(self.destination),
-            fmt_address_for_domain(self.destination, self.recipient),
-            hex::encode(&self.body)
-        )
+        write!(f, "HyperlaneMessage {{ id: {:?} }}", self.id(),)
```

Comment:
can we keep everything but body and version actually? the others are decently useful and we'll probably cut down logging by at least 50%

Reply:
Also I'm a little wary of removing the body for all Debug logs we do - I was hoping we could certainly remove it for the "Popped opqueue" logs but that elsewhere we'd still have it, but happy to give it a try and see what happens. My main concern is just that if we don't have scraper support everywhere it's useful to have the body, but maybe this isn't such a big concern.

If there's an easy way to do what I'm describing where we just omit the body for the popped opqueue log, then that's probably preferable though

Reply:
Updated back to original Debug message minus version and body.
Changed "Popped OpQueue operations" to only show the `operation.id()` instead